.github/workflows/release_subctl_devel.yml

```diff
@@ -1,14 +1,13 @@
 ---
-name: Release Subctl Devel
+name: Release Subctl Cutting Edge
 
 on:
   push:
     branches:
       - devel
-      - release-*
```

Comment:
This doesn't seem necessary since `devel` is replaced by the actual release branch name when the branch is created.